README.md

```diff
@@ -157,7 +157,7 @@ If you want to build the development branch instead, do:
 
     ```shell
     $ git clone https://github.com/Microsoft/vcpkg.git /path/to/vcpkg
-    $ cd /path/to/vcpkg && git checkout 2022.07.25 && ./bootstrap-vcpkg.sh
+    $ cd /path/to/vcpkg && git fetch && git checkout 2022.09.27 && ./bootstrap-vcpkg.sh
```

Comment:
What's the point of git fetch here if you just cloned it above?

Reply:
Ah, just in case you cloned it before you need a fetch or you cannot checkout that tag. I guess people can just use their brains so I will remove it.

Reply:
It's a good point. You can alternatively put this in a comment (# if already cloned, make sure to checkout the correct tag) for above the clone, for instance.